plugins/inputs/open_hardware_monitor/sample.conf

```diff
@@ -0,0 +1,18 @@
+[[inputs.open_hardware_monitor]]
```

Comment:
Please put this sample configuration file in etc\open_hardware_monitor-telegraf-plugin.config 
Makefile script automatically pack it into zip file while generating github release.